taskcluster/client.py

```diff
@@ -496,6 +496,11 @@ def _makeHttpRequest(self, method, route, payload):
             if status == 204:
                 return None
 
+            # Handle content length 0 (empty server responses)
+            servresp = response.length
```

Comment:
Hmm, looks like this doesn't work
```python
(py27) [jhford@jhford-desktop taskcluster-client.py]$ python
Python 2.7.14 (default, Feb 17 2018, 10:42:17) 
[GCC 7.3.1 20180130 (Red Hat 7.3.1-2)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import requests
>>> response = requests.get('http://www.google.com')
>>> response.length
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'Response' object has no attribute 'length'
>>> 
```

I think we might want to look at:
```python
response.headers['Content-Length']
```

Since it's a string, we'll need to parse it into an int.  We should handle the cases that the header isn't there, that it doesn't parse as an int and that it parses as an invalid int (e.g. negative)